### PR TITLE
Script validations.

### PR DESCRIPTION
**Pull request by ludeeus**
### Description:
Validations for included scripts:
 - [X] appdaemon
 - [X] duckdns
 - [X] homeassistant-dev
 - [X] homeassistant
 - [X] homebridge
 - [X] hue
 - [X] libcec
 - [X] mariadb
 - [X] mosquitto
 - [X] mssql
 - [X] mysql
 - [X] postgresql
 - [X] samba
 - [X] tradfri
 - [X] webterminal
***
 - hassbian-script-dev _No need_
 - hassbian-script _No need_
 - hassbian _No need_
 - razberry _No need_

**Related issue (if applicable):** Fixes #<hassbian-scripts issue number goes here>

### Checklist:
  - [X] The code change is tested and works locally.
  - [X] Script has validation check of the job. _This **is** the validation check :)_
  
#### If pertinent:
  - Created/Updated documentation at `/docs` _No need_

### CODE REVIEW

File: package/opt/hassbian/suites/homebridge.sh [hunk 1]
```diff
@@ -128,20 +128,22 @@ fi
...
-	echo "use this: '$HOMEBRIDGE_PIN'"
-	echo "For more information see this repo:"
-	echo "https://github.com/home-assistant/homebridge-homeassistant#customization"
-	echo
+if [ ! -z "${validation}" ]; then
+  echo
+  echo -e "\\e[32mInstallation done.\\e[0m"
+  echo
+  echo "Homebridge is now running and you can add it to your"
+  echo "home-kit app on your iOS device, when you are asked for a pin"
```
Comment (Landrash) [inline package/opt/hassbian/suites/homebridge.sh]: Isn't the app called HomeKit?

Reference: https://www.apple.com/shop/accessories/all-accessories/homekit

**Review by Landrash — CHANGES_REQUESTED** [review]
Looks good with one minor tweak.
Feel free to fix that and well merge this :trophy:

File: package/opt/hassbian/suites/homebridge.sh (hunk 1 above)
Comment (ludeeus) [reply package/opt/hassbian/suites/homebridge.sh]: It its :)

**Review by Landrash — APPROVED** [review]
Looks great!

**Pull request merged**